sdk/FunctionMetadataLoaderExtension/Startup.cs

```diff
@@ -18,17 +18,19 @@ namespace Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader
 {
     internal class Startup : IWebJobsStartup2, IWebJobsConfigurationStartup
     {
-        private const string _workerConfigFile = "worker.config.json";
-        private const string _exePathPropertyName = "defaultExecutablePath";
+        private const string WorkerConfigFile = "worker.config.json";
+        private const string ExePathPropertyName = "defaultExecutablePath";
+        private const string ExeSelfContainedReserved = "{WorkerRoot}";
```

Comment:
This is a nit -- but let's remove reference to `SelfContained` here -- call it `WorkerRootToken` or something. In the loader we don't know about self-contained -- all we know is that we need to replace this if we see it.